prime-router/docs/observability/local-dev-setup.md

```diff
@@ -0,0 +1,41 @@
+# Connect Local Environment to appinsights
+
+## Prerequisites
+ - CDC account 
+ - SU access 
```

Comment:
nit: I would combine this one with the below point and refactor to: `Azure Portal CDC Super User Access`

Reply:
will do